README.md

```diff
@@ -2,126 +2,55 @@
 
 ![Build and Test](https://github.com/Jigsaw-Code/outline-server/actions/workflows/build_and_test_debug.yml/badge.svg?branch=master) [![Mattermost](https://badgen.net/badge/Mattermost/Outline%20Community/blue)](https://community.internetfreedomfestival.org/community/channels/outline-community) [![Reddit](https://badgen.net/badge/Reddit/r%2Foutlinevpn/orange)](https://www.reddit.com/r/outlinevpn/)
 
-This repository has all the code needed to create and manage Outline servers on
-DigitalOcean. An Outline server runs instances of Shadowsocks proxies and
-provides an API used by the Outline Manager application.
+Outline Server is a tool to create and manage your own VPN servers using Shadowsocks proxies. If you prefer a graphical interface, you can use the [Outline Manager](https://github.com/Jigsaw-Code/outline-apps/).
 
-Go to https://getoutline.org for ready-to-use versions of the software. **To join our Outline Community, [sign up for the IFF Mattermost](https://internetfreedomfestival.org/wiki/index.php/IFF_Mattermost).**
+**Components:**
 
-## Components
+- **Outline Server** ([`src/shadowbox`](src/shadowbox)): The core proxy server that runs a Shadowsocks instance. It provides a REST API for access key management.
```

Comment:
Please mention that is uses outline-ss-server as the Shadowsocks backend.

Reply:
Done.